### PR TITLE
Adding substract function

### PR DESCRIPTION
**Pull request by SandroG98**

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -1,2 +1,6 @@
 def add(x, y, z=0):
     return x + y + z
+
+
+def sub(x, y):
```
Comment (danielhuppmann) [inline utils.py:5]: Please add a docstring

Comment (SandroG98) [reply utils.py:5]: Ok i will add it

Comment (danielhuppmann) [reply utils.py:5]: Thank you

**Review by danielhuppmann — CHANGES_REQUESTED** [review]
Please change x

**Review by danielhuppmann — APPROVED** [review]
Thank you

**Pull request merged**